eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -4,10 +4,12 @@
 
     <!-- Ignoring the known prebuilts. -->
     
-    <!-- Tracked in https://github.com/dotnet/source-build/issues/3437 -->
+    <!-- These will go away when repo updates targeting to net8.0 -->
     <UsagePattern IdentityGlob="Microsoft.AspNetCore.App.Ref/7.0.2" />
     <UsagePattern IdentityGlob="Microsoft.NETCore.App.Host.linux-x64/7.0.2" />
     <UsagePattern IdentityGlob="Microsoft.NETCore.App.Ref/7.0.2" />
+
+    <!-- Tracked in https://github.com/dotnet/source-build/issues/3437 -->
```

Comment:
Thoughts on adding these two SBRP right away to avoid the need for a follow-up PR?

Reply:
Yeah - I wasn't sure if we want to add 7.0.2 SBRPs at all. Working on these two right now.

Reply:
`Microsoft.VisualStudio.Setup.Configuration.Interop/3.2.2146` is coming from MSBuild, dependency of `Microsoft.Build.Tasks.Core/17.7.0-preview-23217-02`.

When this repo updates MSBuild dependency, this package might get revved to a newer one, which will necessitate a new SBRP package.

Alternative fix would be to modify the package exclusion to use wildcard for version.